nitro-protocol/test/contracts/examples/SingleAssetPayments/requireStateSupported.test.ts

```diff
@@ -69,18 +67,16 @@ describe('requireStateSupported', () => {
   let channelNonce = getRandomNonce('SingleAssetPayments');
   beforeEach(() => (channelNonce += 1));
   it.each`
-    isValid  | numAssets | isAllocation      | balancesA             | turnNumB | balancesB             | whoSignedWhat     | reason       | description
-    ${true}  | ${[1, 1]} | ${[true, true]}   | ${{A: 1, B: 1, C: 1}} | ${3}     | ${{A: 0, B: 2, C: 1}} | ${whoSignedWhatA} | ${undefined} | ${'A pays B 1 wei'}
-    ${true}  | ${[1, 1]} | ${[true, true]}   | ${{A: 1, B: 1, C: 1}} | ${4}     | ${{A: 1, B: 0, C: 2}} | ${whoSignedWhatB} | ${undefined} | ${'B pays C 1 wei'}
-    ${true}  | ${[1, 1]} | ${[true, true]}   | ${{A: 1, B: 1, C: 1}} | ${5}     | ${{A: 1, B: 2, C: 0}} | ${whoSignedWhatC} | ${undefined} | ${'C pays B 1 wei'}
-    ${false} | ${[1, 1]} | ${[true, true]}   | ${{A: 1, B: 1, C: 1}} | ${5}     | ${{A: 0, B: 2, C: 1}} | ${whoSignedWhatA} | ${reason1}   | ${'A pays B 1 wei (not their move)'}
-    ${false} | ${[1, 1]} | ${[false, false]} | ${{A: 1, B: 1, C: 1}} | ${3}     | ${{A: 0, B: 2, C: 1}} | ${whoSignedWhatA} | ${reason2}   | ${'Guarantee'}
-    ${false} | ${[1, 1]} | ${[true, true]}   | ${{A: 1, B: 1, C: 1}} | ${3}     | ${{A: 1, B: 2, C: 1}} | ${whoSignedWhatA} | ${reason3}   | ${'Total amounts increase'}
-    ${false} | ${[2, 2]} | ${[true, true]}   | ${{A: 1, B: 1, C: 1}} | ${3}     | ${{A: 2, B: 0, C: 1}} | ${whoSignedWhatA} | ${reason4}   | ${'More than one asset'}
+    numAssets | isAllocation      | balancesA       | turnNumB | balancesB       | whoSignedWhat     | reason       | description
```

Comment:
Out of scope nit: `turnNumB` is arguably a legacy parameter. It is no longer a part of the `ForceMove` API, so it is a bit confusing to have test cases defined using it in this way. 

I think it would potentially be more natural to have something like `turnNums: [number,number]`.

Reply:
0dbdc45

Reply:
Agree with you. Applied this suggestion in the commit above.